scenarios_b/08b_Apocalypse.cfg

```diff
@@ -76,6 +76,17 @@
         user_team_name= _ "Wesnoth Expeditionary Force"
         hidden=yes
     [/side]
+    #[side]
+    #    side=5
+    #    {FLAG_VARIANT loyalist}
+    #    controller=ai
+    #    no_leader=yes
+    #    {GOLD 225 275 325}
+    #    recruit=Spearman,Bowman,Cavalryman,Fencer,Horseman,Longbowman,Pikeman,Javelineer,Duelist,Knight
+    #    team_name=evil
+    #    user_team_name= _ "Wesnoth Expeditionary Force"
+    #    hidden=yes
+    #[/side]
```

Comment:
Generally I prefer ifdef-ing unused code out, instead of commenting it out... or, just removing it would be fine, too...